### DISCUSSION TITLE
how to disable WDT?

### DISCUSSION THREAD

**Discussion by circuit-activebuildings** [earlephilhower/arduino-pico#1337] [Q&A] [answered]
I am using watchdog timer but somewhere in my code I need it to be disabled. I tried searching for similar keyword to watchdog_disable() but I did not found that anywhere. So please help me out.

**Comment by Andy2No** [comment, marked as answer]
It seems it can't be stopped, once it's started.  The best you can do is set it to take the maximum 8.3 seconds.   You can also reset the count at any time - do that frequently in loop(), to stop it triggering.

#997 has a link to the documentation, and to a video lecture on the subject by Van Hunter Adams.

**Reply by circuit-activebuildings** [reply]
Ohh that's disappointing... is it yet to included in package or there is no way to disable it in pico like we do in wdt_disable() is arduino boards
I am using 3rd party complex library which I did not wanted to mess with & it takes more than 10seconds to return its sensor value. Anyway Thanks for responding.

**Comment by lyusupov** [comment]
One may try to

```   
    hw_clear_bits(&watchdog_hw->ctrl, WATCHDOG_CTRL_ENABLE_BITS);
``` 
Take a look at Pico SDK  code for more details:
https://github.com/raspberrypi/pico-sdk/blob/master/src/rp2_common/hardware_watchdog/watchdog.c
